examples/manifest.toml

```
@@ -6,105 +6,126 @@
 # To add examples to the app itself, use the `channel` attribute in the example's readme's frontmatter.
 # See <https://github.com/rerun-io/rerun/tree/main/examples#adding-a-new-example> for more information.
 
-[categories.real-data]
+[categories.spatial-computing]
 order = 1
-title = "Examples with Real Data"
+title = "Spatial computing and XR"
 prelude = """
-The following examples illustrate using the Rerun logging SDK with potential real-world (if toy) use cases.
-They all require additional data to be downloaded, so an internet connection is needed at least once.
-Each example downloads it's own data, so no additional steps are needed.
-For the simplest possible examples showing how to use each api,
-check out [Types](/docs/reference/types).
-
-You can try these examples live at [rerun.io/viewer](/viewer).
+Examples related to spatial computing, augmented reality, virtual reality, and mixed reality.
 """
 examples = [
   "arkit_scenes",
-  "controlnet",
-  "depth_guided_stable_diffusion",
-  "detect_and_track_objects",
-  "dicom_mri",
-  "face_tracking",
-  "gesture_detection",
-  "human_pose_tracking",
-  "lidar",
-  "live_camera_edge_detection",
-  "live_depth_sensor",
-  "llm_embedding_ner",
-  "nuscenes",
+  "arflow",
+  "limap",
   "objectron",
-  "open_photogrammetry_format",
-  "raw_mesh",
-  "rgbd",
-  "ros_node",
-  "rrt-star",
-  "segment_anything_model",
-  "signed_distance_fields",
-  "structure_from_motion",
+  "simplerecon",
+  "slahmr",
   "vrs",
+  "widebaseline",
```

arflow -> last position
```suggestion
  "widebaseline",
  "arflow",
```